nahand/main.js

```diff
@@ -6,7 +6,7 @@ let win
 
 function createWindow () {
   // Create the browser window.
-  win = new BrowserWindow({width: 800, height: 600})
+  win = new BrowserWindow({width: 1800, height: 1600})
```

Comment:
That makes sense. I revert this change (I shouldn’t push it to begin with) and I will add an issue about that.